### ISSUE TITLE
Updating an order 'corrupts' the items data

### ISSUE THREAD

**Issue by itsmesaskovic** [damcclean/simple-commerce#649]
### Bug Description

Below is a screenshot of the order in CP before and after an order update:
![image](https://user-images.githubusercontent.com/81617758/172124918-e712d27c-b0cc-44c0-a4bd-3b2f4d4df1bf.png)

Looking in the database we see that the backward slashes are added to the record corrupting it for the order overview in the CP.
![image](https://user-images.githubusercontent.com/81617758/172125108-5dbc7bd3-0045-49f6-b624-fbdb42b27a58.png)

### Steps to reproduce

1. Set items handle to listable in blueprints/orders.yaml
2. Insert a new order from the FE
3. View order overview in SC CP
4. Edit order from CP
5. View order overview in SC CP

### Environment

Statamic 3.3.13 Pro
Laravel 8.83.15
PHP 8.0.19
doublethreedigital/runway 2.3.6
doublethreedigital/simple-commerce 3.2.2
joetannenbaum/antlers-stack-tags 0.1.0
optimoapps/statamic-bard-text-align 1.0.2
roorda-ict/statamic-entries-export 1.0.0

**Comment by duncanmcclean** [comment]
Looks like it's being cast incorrectly when it's saved to the database... are you using a custom Eloquent model or anything?

**Comment by itsmesaskovic** [comment]
You're correct. A custom Order Eloquent model is used. 
The cast is set as follows:
```
    protected $casts = [
        'is_paid' => 'boolean',
        'is_shipped' => 'boolean',
        'is_refunded' => 'boolean',
        'items' => 'json',
        'grand_total' => 'integer',
        'items_total' => 'integer',
        'tax_total' => 'integer',
        'shipping_total' => 'integer',
        'coupon_total' => 'integer',
        'use_shipping_address_for_billing' => 'boolean',
        'gateway' => 'json',
        'data' => 'json',
        'paid_date' => 'datetime',
    ];
```

**Comment by duncanmcclean** [comment]
Hmm, that looks fine. You are casting `items` to JSON (which is what it should be)... Not quite sure what's happening there, I can take another look tonight though.

**Comment by duncanmcclean** [comment]
Managed to reproduce & fix the issue!

I've tagged a release of Runway (the addon which makes the Control Panel work for Eloquent models) which fixes the issue - see [v2.3.8](https://github.com/doublethreedigital/runway/releases/tag/v2.3.8). If you update the version within your site, it should work as expected!
